.travis.yml

```diff
@@ -31,7 +35,7 @@ addons:
     - libvirt-dev
 
 install:
-    - pip install -r requirements-travis.txt
```

Comment:
@cacarrara the one issue I see here is: we want the CI environment to be as stable as possible, and for that, we don't want the latest version of requirements to be installed when one is released.  To not keep duplicate copies, I'd be OK with getting rid of the "requirement-travis.txt" file, and determining that to run tests, all deps will be pinned to a given version.

Developers can choose to either install those exact versions, use the version provided on their distros, or use the latest and greatest.  TBH, we shouldn't care what version developer have.  But, we should definitely control the versions on CI.